12-connectivity.tf

```diff
@@ -194,13 +169,12 @@ resource "azurerm_subnet_route_table_association" "application_gateway_subnet" {
 }
 
 resource "azurerm_route" "additional_route_appgw" {
-  for_each = { for route in var.additional_routes_appgw : route.name => route }
```

Comment:
CFT is using this value `additional_routes_appgw` so it needs a PR to migrate it

Reply:
reference and TFVar names have been updated to `additional_routes_appgw`